packages/pluggableWidgets/slider-web/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "slider-web",
   "widgetName": "Slider",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
RC is not released yet, so you can leave this as 2.0.0

Reply:
@Keraito, reverted